momepy/functional/_elements.py

```diff
@@ -15,6 +15,7 @@
 GPD_GE_013 = Version(gpd.__version__) >= Version("0.13.0")
 GPD_GE_10 = Version(gpd.__version__) >= Version("1.0dev")
 LPS_GE_411 = Version(libpysal.__version__) >= Version("4.11.dev")
+SHPLY_GE_205 = Version(shapely.__version__) >= Version("2.0.5")
```

Comment:
This shall check for shapely 2.1. There may be 2.0.5 or 2.0.6 none of which will include it. Shapely in the dev CI env reports `shapely-2.1.0.dev0+83.ge9be991`